src/Frontend/Core/Engine/Frontend.php

```diff
@@ -43,7 +43,10 @@ public function display()
     public function initialize()
     {
         new Url($this->getKernel());
-        new Template();
+        $templateType = (string) $this->getContainer()->getParameter('fork.template');
```

Comment:
Is the cast to string needed here? You don't use a strict comparison (`===`) to check the variable later, so I don't think it's needed to cast this here.

Even if you use a strict comparison, the getParameter should return a string (or null)